packages/contracts/contracts/minter-suite/Minters/MinterSEAV1.sol

```diff
@@ -0,0 +1,1215 @@
+            _sender: msg.sender
+        });
+        MaxInvocationsLib.manuallyLimitProjectMaxInvocations(
+            _projectId,
+            _coreContract,
+            _maxInvocations,
+            _maxInvocationsProjectConfigMapping[_coreContract][_projectId]
+        );
+        emit ProjectMaxInvocationsLimitUpdated(
+            _projectId,
+            _coreContract,
+            _maxInvocations
+        );
+
+        // @dev the following is unique behavior to this minter
+        // for convenience, try to mint and assign a token to the project's
+        // next slot
+        _tryMintTokenToNextSlot(_projectId, _coreContract);
+    }
+
+    /**
+     * @notice Sets the minter-wide time buffer in seconds. The time buffer is
+     * the minimum amount of time that must pass between the final bid and the
+     * the end of an auction. Auctions are extended if a new bid is placed
+     * within this time buffer of the auction end time.
+     * @param _minterTimeBufferSeconds Time buffer in seconds.
+     */
+    function updateMinterTimeBufferSeconds(
+        uint32 _minterTimeBufferSeconds
+    ) external {
+        // CHECKS
+        _onlyNonZero(_minterTimeBufferSeconds);
+        AuthLib.onlyMinterFilterAdminACL({
+            _minterFilterAddress: minterFilterAddress,
+            _sender: msg.sender,
+            _contract: address(this),
+            _selector: this.updateMinterTimeBufferSeconds.selector
+        });
+        // EFFECTS
+        minterTimeBufferSeconds = _minterTimeBufferSeconds;
+        emit MinterTimeBufferUpdated(_minterTimeBufferSeconds);
+    }
+
+    /**
+     * @notice Sets the gas limit during ETH refunds when a collector is
+     * outbid. This value should be set to a value that is high enough to
+     * ensure that refunds are successful for commonly used wallets, but low
+     * enough to avoid excessive abuse of refund gas allowance during a new
+     * bid.
+     * @dev max gas limit is ~16M, which is considered well over a future-safe
+     * upper bound.
+     * @param _minterRefundGasLimit Gas limit to set for refunds. Must be
+     * between 7,000 and max uint24 (~16M).
+     */
+    function updateRefundGasLimit(uint24 _minterRefundGasLimit) external {
+        // CHECKS
+        AuthLib.onlyMinterFilterAdminACL({
+            _minterFilterAddress: minterFilterAddress,
+            _sender: msg.sender,
+            _contract: address(this),
+            _selector: this.updateRefundGasLimit.selector
+        });
+        // @dev max gas limit implicitly checked by using uint16 input arg
+        // @dev min gas limit is based on rounding up current cost to send ETH
+        // to a Gnosis Safe wallet, which accesses cold address and emits event
+        require(_minterRefundGasLimit >= 7_000, "Only gte 7_000");
+        // EFFECTS
+        minterRefundGasLimit = _minterRefundGasLimit;
+        emit MinterRefundGasLimitUpdated(_minterRefundGasLimit);
+    }
```

Comment:
Curious if we want to make this a constant to simplify and reduce need for spurious minterfilter admin gates?

Reply:
I *think* it is still worth making this admin-configurable on the first iteration of this minter. Until we release this into the wild, I'm hesitant to hard-code this value. Currently, making it configurable only adds this function - events still need to be emitted in constructor, handled in subgraph, etc.
If you have stronger feelings on this, happy to discuss more!

Reply:
I don't feel super strongly here – if we think its worth having this flexibility, SGTM to keep it included.